packages/ui/src/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { useState } from 'react'
+import { Checkbox, CheckedState } from './checkbox'
+
+const meta: Meta<typeof Checkbox> = {
+  title: 'ui/Checkbox',
+  component: Checkbox,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    disabled: {
+      control: 'boolean',
+    },
+  },
+}
+
+export default meta
+
+export const Default: StoryObj<typeof Checkbox> = {
+  render(props) {
+    return (
+      <div className="flex gap-4">
+        <Checkbox {...props} defaultChecked={false} />
+        <Checkbox {...props} defaultChecked />
+        <Checkbox {...props} defaultChecked="indeterminate" />
+      </div>
+    )
+  },
+}
+
+export const SizeLg: StoryObj<typeof Checkbox> = {
+  render() {
+    return (
+      <div className="flex gap-4">
+        <Checkbox size="lg" defaultChecked={false} />
+        <Checkbox size="lg" defaultChecked />
+        <Checkbox size="lg" defaultChecked="indeterminate" />
+      </div>
+    )
+  },
+}
+
+export const WithControll: StoryObj<typeof Checkbox> = {
+  render() {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Please refer to the comment to remove the line - https://github.com/storybookjs/storybook/issues/22593#issuecomment-1552286276

Reply:
<img width="1134" alt="image" src="https://github.com/consolelabs/websites/assets/12707960/4ef76ef8-5af3-4de1-91c1-1e3cc37a2cff">